schemas/get-worker-type-response.yml

```diff
@@ -0,0 +1,112 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-worker-type-response.json#
```

Comment:
They are subtly different.  The get has a workerType property where the create doesn't have this property.  I guess I could have the property in both, but make an assertion that the `:workerType` param === the one from the body of the document...

I'll keep this in mind as something to fix in a future iteration.

For now, I have rewritten the schema in a much more precise way.